### ISSUE TITLE
Allow configuration discussion to be moved to end

### ISSUE THREAD

**Issue by erica** [apple/swift-argument-parser#159]
I know this one is a little tenuous but I thought I'd throw it out there and if you don't like it, it can be as easily closed as any other feature request:

A flag that allows the discussion text (`CommandConfiguration(discussion: "")`) to be displayed at the end of help instead of the beginning so the options are visible to the user in the first screen of `more`/`less`. The current output is heavily weighted towards the style of internal code comments instead of the tooling needs of the end-user, who often wants to review the flags, options, arguments (and okay subcommands) quickly, and then use the tool immediately. This is the pattern used for memory refresh.

Reading the full discussion is more typical for someone who is encountering an issue with the tooling and looking for deeper instruction material, and is more typically secondary to casual use. (see also: "full help")

**Comment by natecook1000** [comment]
Agreed on this, but I think I might want to have three slots available, so it ends up matching the way `man` pages are often layed out:

```
$ example --help
Usage: example --foo [bar]
Abstract
Discussion
Argument/flag/option descriptions
Additional info
```

@erica Do you have any suggestions for a name for “Additional info” here?

**Comment by natecook1000** [comment]
Name for this extra configuration property can be `extendedDiscussion`.

**Comment by arthurcro** [comment]
Hey! :wave: Are you still interested in closing issue? I'd love to give this a try!

**Comment by natecook1000** [comment]
That would be great, thanks @arthurcro!
